hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/SingleTransactionRecordBuilderImpl.java

```diff
@@ -609,6 +616,32 @@ public SingleTransactionRecordBuilderImpl addTokenTransferList(@NonNull final To
         return this;
     }
 
+    /**
+     * Sets the pending airdrop record list
+     *
+     * @param pendingAirdropRecords the pending airdrop record list
+     * @return the builder
+     */
+    @Override
+    public TokenAirdropRecordBuilder pendingAirdrops(@NonNull List<PendingAirdropRecord> pendingAirdropRecords) {
+        requireNonNull(tokenTransferLists, "tokenTransferLists must not be null");
```

Comment:
Is this a typo ? Why do we require tokenTransferLists

Reply:
Yes, fixed